Lib/test/test_httpservers.py

```diff
@@ -17,8 +17,11 @@
 import urllib.parse
 import html
 import http.client
+import email.message
```

Comment:
I changed the order in commit 	db46671. I tried several orders but couldn't find something obvious. I prefer to leave Internet-related modules together, and `time` and `datetime` together.

... and thanks for the expression "style nit", I didn't know it :-)